src/yall.mjs

```diff
@@ -3,27 +3,31 @@ export default function (options) {
     options = {};
   }
 
-  const intersectionObserverSupport = "IntersectionObserver" in window && "IntersectionObserverEntry" in window;
+  const doc = document;
+  const win = window;
+  const arr = [];
+  const qsa = (root, sel) => arr.slice.call(root.querySelectorAll(sel));
+
+  const intersectionObserverSupport = "IntersectionObserver" in win && "IntersectionObserverEntry" in win;
   const lazyClass = options.lazyClass || "lazy";
   const lazyBackgroundClass = options.lazyBackgroundClass || "lazy-bg";
   const idleLoadTimeout = "idleLoadTimeout" in options ? options.idleLoadTimeout : 100;
   const observeChanges = options.observeChanges || false;
   const selectorString = `img.${lazyClass},video.${lazyClass},iframe.${lazyClass},.${lazyBackgroundClass}`;
 
-  // This abstraction shaves off a few bytes (plus it's nifty).
-  const sliceCall = arr => [].slice.call(arr);
```

Comment:
NodeLists aren't iterable in IE11 using forEach :\ There may be some further opportunities here, though to see if maybe we can iterate over them with `for` instead. In some cases, I've found that going from `forEach` to `for` can lop off a few bytes.

Reply:
Wow, I totally did not read this right. I see what you did here and this makes a lot of sense. Nevermind that first comment. I'm totally bringing this in.

Reply:
yeah, it's still doing the same thing.

Reply:
heckin' smart!

Reply:
nah, i'm a pretty average golfer. you should see the crazy shit the Preact team has to do to shave every byte off.

also:

https://js1k.com/
https://www.dwitter.net/

Reply:
I'm a reasonably OK, but amateur golfer, but I've looked at the PRs for preact. It's all shit like "-4B" but it all adds up.

omg TIL about dwitter. This is so cool, thanks for sharing it!